### ISSUE TITLE
Train&Test set split

### ISSUE THREAD

**Issue by XingzhuoYan** [azamshoaib/DRFuser#2]
Thanks for your last reply. I have one more question: Could you please tell me how the train_dataset and test_set is splitted? 
I see you saved the two sets in different data folders but I don't know the rule of the seperation. 

And I cann't find the test.sh file and got the error: "/home/shoaib/work/dvs_data/test.sh: no such file". Could you please tell me where I can find this file? Thanks a lot for your help.

**Comment by azamshoaib** [comment]
@XingzhuoYan Thank you for using the code. We have adopted the standard train and test data split but you can also change it for your work. 
For your second question, this test.sh file is just to source the ROS, you can comment it for your settings. As in our settings, we  have to source the ROS environment so that imports related to ROS works. However for the ease you can write the following in your test.sh 
`source path to the workspace/devel/setup.bash`
I hope that clarifies your queries. If you have any question, kindly do let me know. Thank you.
